Can't Delete Portals in DNN 9.2.1

<!-- 
  Please read contribution guideline first: https://github.com/dnnsoftware/Dnn.Platform/blob/development/CONTRIBUTING.md 
  Any potential security issues should be sent to security@dnnsoftware.com, rather than posted on GitHub
-->

## Description of bug
Deleting a portal (Persona Bar "Sites") doesn't work.  

## Steps to reproduce
Go to Persona Bar, Manage, Sites.
Mouseover a site and click the trashcan icon to delete.

## Current result
A generic error pops up, nothing is logged in the Event Log, but an error message appears in the xxx.log.resources file:

2018-10-01 11:17:37.780-04:00 [Surface-Joe][D:2][T:15][ERROR] Dnn.PersonaBar.Sites.Services.SitesController - System.UnauthorizedAccessException: Access to 'C:\Users\Joe\Documents\Websites\DNN921-test\Website' denied. ---> System.ComponentModel.Win32Exception: Access is denied
   --- End of inner exception stack trace ---
   at SchwabenCode.QuickIO.Internal.InternalQuickIOCommon.NativeExceptionMapping(String path, Int32 errorCode)
   at SchwabenCode.QuickIO.Internal.InternalQuickIO.GetFindDataFromPath(QuickIOPathInfo pathInfo)
   at DotNetNuke.Common.Utilities.FileSystemUtils.DeleteFilesRecursive(String strRoot, String filter)
   at DotNetNuke.Entities.Portals.PortalController.DeletePortal(PortalInfo portal, String serverPath)
   at Dnn.PersonaBar.Sites.Services.SitesController.DeletePortal(Int32 portalId)


## Expected result
Site is deleted

## Screenshots
If applicable, provide screenshots to help explain your problem.

## Error log
There is no Event Log entry.
The error in xx.log.resources is:

the xxx.log.resources file:

2018-10-01 11:17:37.780-04:00 [Surface-Joe][D:2][T:15][ERROR] Dnn.PersonaBar.Sites.Services.SitesController - System.UnauthorizedAccessException: Access to 'C:\Users\Joe\Documents\Websites\DNN921-test\Website' denied. ---> System.ComponentModel.Win32Exception: Access is denied
   --- End of inner exception stack trace ---
   at SchwabenCode.QuickIO.Internal.InternalQuickIOCommon.NativeExceptionMapping(String path, Int32 errorCode)
   at SchwabenCode.QuickIO.Internal.InternalQuickIO.GetFindDataFromPath(QuickIOPathInfo pathInfo)
   at DotNetNuke.Common.Utilities.FileSystemUtils.DeleteFilesRecursive(String strRoot, String filter)
   at DotNetNuke.Entities.Portals.PortalController.DeletePortal(PortalInfo portal, String serverPath)
   at Dnn.PersonaBar.Sites.Services.SitesController.DeletePortal(Int32 portalId)


## Additional context
Instead, I get a generic error box popping up over the Personal Bar, nothing in the Event Log, but this in 
-------------------------------------------
I can delete portals in 8.0.4.

In poking around, deleting a portal (8.0.4) doesn't delete a lot of stuff in most of the portal related tables.  I think that it should ...

## Affected version
<!-- Check all that apply and add more if necessary -->

* [] 9.2.2
* [x] 9.2.1
* [] 9.2
* [] 9.1.1
* [ ] 9.1
* [ ] 9.0

## Affected browser
N/A

* [ ] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Internet Explorer
* [ ] Edge

--- a/DNN Platform/Library/Common/Utilities/FileSystemUtils.cs
+++ b/DNN Platform/Library/Common/Utilities/FileSystemUtils.cs
@@ -467,7 +467,7 @@ namespace DotNetNuke.Common.Utilities
                             DeleteFilesRecursive(strFolder, filter);
                         }
                     }
-                    foreach (string strFile in Directory.EnumerateFilePaths(new DirectoryInfo(strRoot)).Where(f => f.Contains(filter)))
+                    foreach (string strFile in Directory.EnumerateFilePaths(strRoot).Where(f => f.Contains(filter)))
                     {
                         try
                         {

